src/main/java/com/amazon/pvar/tspoc/merlin/solver/flowfunctions/BackwardFlowFunctions.java

```diff
@@ -628,4 +628,28 @@ private void handleflowToFunctionEntry(ConstantNode entryNode, Value queryVal,
         }
     }
 
+    @Override
+    public void handleUnresolvedCall() {
+        assert containingSolver != null;
+        final var node = context.currentPDSNode().stmt().getNode();
+        if (node instanceof CallNode callNode) {
```

Comment:
Just a thought: is it worth adding the solver and the call node as arguments? By moving these checks to the caller, you're avoiding having to make them here.